package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "axios": "^0.24.0",
     "dotenv": "^10.0.0",
+    "moment": "^2.29.1",
```

Comment:
It's always worth calling out whenever a new dependency is added so we can make sure to run `npm install` afterward